sdk/core/perf/test/CMakeLists.txt

```diff
@@ -38,8 +38,16 @@ target_include_directories(
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/inc>
 )
 
+if(DEFINED ENV{VCPKG-AZURE-CORE-CPP})
```

Comment:
I take it this gets set by the pipeline yml file?

Are there circumstances where you would expect this to be set by SDK authors?

Reply:
nope